### PR TITLE
regenerate

### PR DESCRIPTION
**Pull request by jonathaningram**

### CODE REVIEW

File: generate/generate.sh [hunk 1]
```diff
@@ -12,12 +12,17 @@ function die() {
 if [ -z "$GEN_PATH" ]; then
     die "GEN_PATH must be set, e.g. /path/to/sajari/sdk-python"
 fi
+if [ -z "$TEMPLATES_PATH" ]; then
+    die "TEMPLATES_PATH must be set, e.g. /path/to/sajari/sdk-python/generate/templates"
+fi
+
 VERSION=5.1.0
```
Comment (jkaho) [inline generate/generate.sh]: Should we bump the version too?

**Review by jkaho — DISMISSED** [review]
RCFV

File: docs/SchemaApi.md [hunk 2]
```diff
@@ -342,7 +379,7 @@ with sajari_client.ApiClient(configuration) as api_client:
     # and optional values
     try:
         # List schema fields
-        api_response = api_instance.list_schema_fields(collection_id, page_size=page_size, page_token=page_token)
+        api_response = api_instance.list_schema_fields(collection_id, parent=parent, account_id=account_id, page_size=page_size, page_token=page_token)
```
Comment (jkaho) [inline docs/SchemaApi.md:382]: This isn't a breaking change?

**Review by jkaho — DISMISSED** [review]
RCFV

File: docs/SchemaApi.md (hunk 2 above)
Comment (jonathaningram) [reply docs/SchemaApi.md:382]: Nope because they are optional named arguments. Callers don't have to pass them. And in Python they are in any order too.

**Review by jkaho — APPROVED** [review]

**Pull request merged**